docker-compose-sanitize.rb

```diff
@@ -1,18 +1,37 @@
+require 'json'
 require 'yaml'
 
+universal_options = ["volumes"]
```

Comment:
I'd consider this a constant, as such it probably makes sense to `UNIVERSAL_OPTIONS`